engine/src/events/event_manager.h

```diff
@@ -33,22 +34,21 @@ namespace Bald {
 
         /**
         * @fn       EventManager
-        * @brief    Constructor is deleted because this class is just a system,
-        *           meaning you cannot instantiate it
+        * @brief    Constructor of EventManager class and performs counting of existing objects
```

Comment:
Change it: EventManager constructor. Its role is to increase the static reference counter variable